queries/aws_queries/anomalous_vpc_port_activity.yml

```diff
@@ -0,0 +1,19 @@
+AnalysisType: scheduled_query
+Description: Look for suspected port scanning activity
+Enabled: false
+Query: |
+    SELECT
+      srcAddr,
```

Comment:
For complex VPCs, I suggest adding `region` `vpcId` and `subNetId` to the output and `GROUP BY` clauses.  srcAddr/dstAddr is unlikely to be unique in environments with lots of addresses/overlapping address space